proposals/lip-0065.md

```diff
@@ -187,12 +189,12 @@ The event `topics` is an array of unique byte values whose length ranges between
 
 By default, the first topic is set to:
 
-* the transaction ID, for events emitted during the transaction processing stages,
-* the cross-chain message ID, for events emitted during the cross-chain message processing stages,
 * the constant `EVENT_TOPIC_INIT_GENESIS_STATE`, for events emitted during the 'genesis state initialization' stage,
 * the constant `EVENT_TOPIC_FINALIZE_GENESIS_STATE`, for events emitted during the 'genesis state finalization' stage,
 * the constant `EVENT_TOPIC_BEFORE_TRANSACTIONS`, for events emitted during the 'before transactions execution' stage,
-* or the constant `EVENT_TOPIC_AFTER_TRANSACTIONS`, for events emitted during the 'after transactions execution' stage.
+* the constant `EVENT_TOPIC_AFTER_TRANSACTIONS`, for events emitted during the 'after transactions execution' stage,
+* the constant `EVENT_TOPIC_TRANSACTION_EXECUTION` concatenated with the transaction ID, for events emitted during the transaction processing stages,
+* the constant `EVENT_TOPIC_CCM_EXECUTION` concatenated with the cross-chain message ID, for events emitted during the cross-chain message processing stages.
 
 An event emitted during transaction processing cannot have any of these default values in the `topics` array.
 
```

Comment:
Link of line 266 does not exist, should be replaced by "https://github.com/LiskHQ/lips/blob/main/proposals/lip-0053.md#cross-chain-command-processing-stages"

Reply:
fixed, thanks